spec/features/post_spec.rb

```diff
@@ -0,0 +1,109 @@
+require 'rails_helper'
+
+RSpec.describe 'Post pages test', type: :feature do
+  describe 'index page' do
+    before :each do
+      @user_first = User.create(name: 'Tom', photo: 'tom.png', bio: 'I am Tom', post_counter: 0)
+
+      @post_first = @user_first.posts.new(title: 'Niklaus', text: 'This is my first post')
+      @post_first.likes_counter = 0
+      @post_first.comments_counter = 0
+      @post_first.save
+      @post_first.update_users_posts_counter
+
+      @comment = @user_first.comments.new(post_id: @post_first.id, author_id: @user_first.id, text: 'My first comment')
+      @comment.post_id = @post_first.id
+      @comment.save
+      @comment.update_posts_comments_counter
+
+      @like = Like.new(author_id: @user_first.id, post_id: @post_first.id)
+      @like.author = @user_first
+      @like.post = @post_first
+      @like.save
+      @like.update_posts_likes_counter
+    end
+
+    it 'Should set the link of the picture to the img src' do
+      visit user_posts_path(@user_first.id)
```

Comment:
* Kindly update your tests as some scenarios are missing below 👇 is a screenshot of the missing tests required 
![image](https://github.com/Ogaga01/Blog-App/assets/52497246/0c14f763-f77e-413d-91ef-0b13bc3191e4)